src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1611,6 +1612,37 @@ class spell_dru_wild_growth_aura : public AuraScript
     }
 };
 
+// 384667 - Sudden Ambush Talent
+class spell_dru_sudden_ambush : public AuraScript
+{
+    PrepareAuraScript(spell_dru_sudden_ambush);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_SUDDEN_AMBUSH });
+    }
+
+    bool CheckProc(AuraEffect const* /*aurEff*/, ProcEventInfo& eventInfo)
+    {
+        // Get the Combo Points
+        Spell const* procSpell = eventInfo.GetProcSpell();
+        std::optional<int32_t> comboPointsAmount = procSpell->GetPowerTypeCostAmount(POWER_COMBO_POINTS);
+
+        if (!*comboPointsAmount) {
+            return false;
+        }
+
+        uint8 chances = roll_chance_f(*comboPointsAmount * 5);
+      
+        return chances;
```

Comment:
```suggestion
        return roll_chance_f(*comboPointsAmount * 5);;
```
Might as well return this immediately instead of creating a variable for it